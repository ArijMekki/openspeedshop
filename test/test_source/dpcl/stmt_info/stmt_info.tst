#! /bin/tcsh
./ext1 d path $cwd/mutatee
