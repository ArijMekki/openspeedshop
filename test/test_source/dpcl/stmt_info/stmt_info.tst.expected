bcreate() was successful.
Module name mutatee.c examined
Module mutatee.c found... getting statement info
bget_all_statements() SUCCESS! - status ASC_success
stmt info count 57
stmt line  count 1
   line 6 column 0
stmt address  count 2
stmt line  count 1
   line 6 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 6 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 7 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 11 column 0
stmt address  count 2
stmt line  count 1
   line 11 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 12 column 0
stmt address  count 2
stmt line  count 1
   line 13 column 0
stmt address  count 2
stmt line  count 1
   line 13 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 382 column 0
stmt address  count 2
stmt line  count 1
   line 382 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 317 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 16 column 0
stmt address  count 2
stmt line  count 1
   line 16 column 0
stmt address  count 2
stmt line  count 1
   line 16 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 15 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
stmt line  count 1
   line 9 column 0
stmt address  count 2
bget_all_statements() SUCCESS !!!!

